src/Pages/Store/Category/Component/CategoryContent/Component/ContentFilter/Component/Component/FilterList.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import "./FilterList.scss";
+
+class FilterList extends Component {
+  state = {
+    checked: false,
+    selectOption: this.props.defaultChecked,
+  };
```

Comment:
props를 바로 받아와서 state에 넣는 경우라면 다음과 같이 할 수도 있습니다!
```suggestion
  constructure(props){
    super(props);
    this.state = {
      checked: false,
      selectOption: props.defaultChecked,
    }
  };
```